test/index.test.js

```diff
@@ -110,30 +110,39 @@ describe('mongodb-connection-model', function() {
     });
 
     describe('ATLAS - mongodb.net', function() {
-      var atlasConnection = 'mongodb://ADMINUSER:PASSWORD@' +
+      var atlasConnection = 'mongodb://ADMINUSER:<PASSWORD>@' +
           'a-compass-atlas-test-shard-00-00-vll9l.mongodb.net:38128,' +
           'a-compass-atlas-test-shard-00-01-vll9l.mongodb.net:38128,' +
-          'a-compass-atlas-test-shard-00-02-vll9l.mongodb.net:38128/admin?' +
+          'a-compass-atlas-test-shard-00-02-vll9l.mongodb.net:38128/<DATABASE>?' +
```

Comment:
I think we need to handle `<DATABASE>` as a special case as well. Will talk to you in person.